lib/merchant_repository.rb

```diff
@@ -25,21 +29,21 @@ def find_by_id(id)
 
   def find_by_name(name)
     @merchants.find do |merchant|
-      merchant.name == name
+      merchant.name.downcase == name.downcase
     end
   end
 
   def find_all_by_name(name)
     @merchants.find_all do |merchant|
-      merchant.name == name
+      merchant.name.downcase.include?(name.downcase)
```

Comment:
⛏ Would it be worth using .casecmp here?

Reply:
Love it. Will update.

Reply:
Oh, this actually can't be a casecmp -- it's not comparing like to like, but needs to be able to handle fragments, which I don't think casecmp can do?